Module 7/ex2/Includes/Array.hpp

```diff
@@ -0,0 +1,67 @@
+#ifndef ARRAY_HPP
+#define ARRAY_HPP
+
+#include <iostream>
+#include <stdexcept>
+#include <memory>
+
+template <typename T>
+class Array
+{
+private:
+  std::unique_ptr<T[]> _data;
+  unsigned int _size;
+
+public:
+  Array() : _data(nullptr), _size(0) {}
+
+  Array(unsigned int n) : _data(std::make_unique<T[]>(n)), _size(n)
+  {
+    for (unsigned int i = 0; i < _size; ++i)

```

Comment:
In your opinion, is this loop necessary ?

Reply:
It was just a question here to make you think about it, the change was not required, as it is not bad to have it clearly indicated :)